src/ui/common/src/components/tables/CheckTableItem.tsx

```diff
@@ -1,71 +1,31 @@
-import {
-  faCircleCheck,
-  faCircleExclamation,
-  faMinus,
-  faTriangleExclamation,
-} from '@fortawesome/free-solid-svg-icons';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import Box from '@mui/material/Box';
 import React from 'react';
 
-import { theme } from '../../styles/theme/theme';
-import { stringToExecutionStatus } from '../../utils/shared';
+import ExecutionStatus from '../../utils/shared';
 import { StatusIndicator } from '../workflows/workflowStatus';
 
 interface CheckTableItemProps {
-  checkValue: string;
-  status?: string;
+  status: ExecutionStatus;
+  value?: string;
 }
 
 export const CheckTableItem: React.FC<CheckTableItemProps> = ({
-  checkValue,
   status,
+  value,
 }) => {
-  let iconColor = theme.palette.black;
-  let checkIcon = faMinus;
+  if (value) {
+    return <>{value}</>;
+  }
 
-  if (checkValue) {
-    switch (checkValue.toLowerCase()) {
-      case 'true': {
-        checkIcon = faCircleCheck;
-        iconColor = theme.palette.Success;
-        break;
-      }
-      case 'false': {
-        checkIcon = faCircleExclamation;
-        iconColor = theme.palette.Error;
-        break;
-      }
-      case 'warning': {
-        checkIcon = faTriangleExclamation;
-        iconColor = theme.palette.Warning;
-        break;
-      }
-      case 'none': {
-        checkIcon = faMinus;
-        iconColor = theme.palette.black;
-        break;
-      }
-      default: {
-        // None of the icon cases met, just fall through and render table value.
-        return <>{checkValue}</>;
-      }
-    }
-  } else {
-    // Check value not found, render the status indicator for this check.
-    return (
-      <StatusIndicator
-        status={stringToExecutionStatus(status)}
-        size={'16px'}
-        monochrome={false}
-      />
-    );
+  if (status) {
```

Comment:
This can be combined with the return statement below as a ternary to check for status.

```status={status ? status : ExecutionStatus.Unknown}```